base/applications/rapps/available.cpp

```diff
@@ -231,11 +261,20 @@ AvailableStrings::AvailableStrings()
     //FIXME: maybe provide a fallback?
     if (GetStorageDirectory(szPath))
     {
-        szAppsPath = szPath + L"\\rapps\\";
+        szAppsPath = szPath;
+        PathAppendW(szAppsPath.GetBuffer(MAX_PATH), L"rapps");
+        PathAddBackslashW(szAppsPath.GetBuffer());
```

Comment:
It's because some other places are still using the original way to handle path, without this backslash will lead to some bugs. I haven't change them all, so I add this temporary.